<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="2" ignored="0" total="6" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="13414" started-at="2022-07-07T17:06:14Z" finished-at="2022-07-07T17:06:27Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="13414" started-at="2022-07-07T17:06:14Z" finished-at="2022-07-07T17:06:27Z">
      <class name="com.orangehrm.tests.LoginTests">
        <test-method status="PASS" signature="startReport()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="startReport" is-config="true" duration-ms="310" started-at="2022-07-07T17:06:14Z" finished-at="2022-07-07T17:06:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="setUp" is-config="true" duration-ms="9983" started-at="2022-07-07T17:06:14Z" finished-at="2022-07-07T17:06:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="m1()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="m1" duration-ms="19" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.orangehrm.tests.LoginTests.m1(LoginTests.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="getResult" is-config="true" duration-ms="19" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=m1 status=FAILURE method=LoginTests.m1()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="SKIP" signature="m2()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="m2" duration-ms="0" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[test]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: test
at com.orangehrm.tests.LoginTests.m2(LoginTests.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m2 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="getResult" is-config="true" duration-ms="0" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=m2 status=SKIP method=LoginTests.m2()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="FAIL" signature="m3()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="m3" duration-ms="1" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.orangehrm.tests.LoginTests.m3(LoginTests.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m3 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="getResult" is-config="true" duration-ms="3" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=m3 status=FAILURE method=LoginTests.m3()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="m4()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="m4" duration-ms="8" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m4 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="getResult" is-config="true" duration-ms="0" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=m4 status=SUCCESS method=LoginTests.m4()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="m5()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="m5" duration-ms="0" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m5 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="getResult" is-config="true" duration-ms="1" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=m5 status=SUCCESS method=LoginTests.m5()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="verifyLogin()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="verifyLogin" duration-ms="2239" started-at="2022-07-07T17:06:24Z" finished-at="2022-07-07T17:06:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="getResult" is-config="true" duration-ms="2" started-at="2022-07-07T17:06:26Z" finished-at="2022-07-07T17:06:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLogin status=SUCCESS method=LoginTests.verifyLogin()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="tearDown" is-config="true" duration-ms="1069" started-at="2022-07-07T17:06:26Z" finished-at="2022-07-07T17:06:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.orangehrm.tests.LoginTests@63a65a25]" name="endReport" is-config="true" duration-ms="639" started-at="2022-07-07T17:06:27Z" finished-at="2022-07-07T17:06:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.orangehrm.tests.LoginTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
